docs/configuration/index.md

```diff
@@ -1872,6 +1872,7 @@ The Druid SQL server is configured through the following properties on the Broke
 |`druid.sql.planner.metadataSegmentPollPeriod`|How often to poll coordinator for published segments list if `druid.sql.planner.metadataSegmentCacheEnable` is set to true. Poll period is in milliseconds. |60000|
 |`druid.sql.planner.authorizeSystemTablesDirectly`|If true, Druid authorizes queries against any of the system schema tables (`sys` in SQL) as `SYSTEM_TABLE` resources which require `READ` access, in addition to permissions based content filtering.|false|
 |`druid.sql.planner.useNativeQueryExplain`|If true, `EXPLAIN PLAN FOR` will return the explain plan as a JSON representation of equivalent native query(s), else it will return the original version of explain plan generated by Calcite. It can be overridden per query with `useNativeQueryExplain` context key.|false|
+|`druid.sql.planner.forceExpressionVirtualColumns`|If true, the query plan does not contain any specialized virtual columns. Instead, specialized virtual columns (such as those used to plan `MV_FILTER_ONLY` and `MV_FILTER_NONE` SQL functions) are planned as "expression"-type virtual columns. If false, such virtual columns are planned in their respective specialized forms.|false|
```

Comment:
I sort of wonder if we should _actually_ document this since it seems primarily useful for us to experiment with and I can't think of a very legitimate production use case, but otoh :shrug:

Reply:
+1. we need not document this.

Reply:
Removed.